base/float.jl

```diff
@@ -218,7 +218,7 @@ function ieee754_representation(
 end
 
 """
-    uabs(x::Integer)
+    Base.uabs(x::Integer)
```

Comment:
Why add `Base.` here?

Reply:
But didn't you also add this to the set of exported symbols or am I confused?

Reply:
I was trying to add it to the set of `public` but not exported symbols.

Reply:
But unless I am mistaken, this PR also adds it to `base/export.jl` into the export list, or?